hassio/src/dialogs/snapshot/dialog-hassio-snapshot.ts

```diff
@@ -192,46 +198,36 @@ class HassioSnapshotDialog extends LitElement {
         ${this._error ? html` <p class="error">Error: ${this._error}</p> ` : ""}
 
         <div>Actions:</div>
+        <div class="button-row">
+          ${this._snapshot.type === "full"
+            ? html`
+                <mwc-button @click=${this._fullRestoreClicked}>
+                  <ha-svg-icon .path=${mdiHistory} class="icon"></ha-svg-icon>
+                  Restore Everything
+                </mwc-button>
+              `
+            : ""}
+          <mwc-button @click=${this._partialRestoreClicked}>
+            <ha-svg-icon .path=${mdiHistory} class="icon"></ha-svg-icon>
+            Restore Selected
+          </mwc-button>
+        </div>
         ${!this._onboarding
-          ? html`<mwc-button
-              @click=${this._downloadClicked}
-              slot="primaryAction"
-            >
-              <ha-svg-icon .path=${mdiDownload} class="icon"></ha-svg-icon>
-              Download Snapshot
-            </mwc-button>`
-          : ""}
-
-        <mwc-button
-          @click=${this._partialRestoreClicked}
-          slot="secondaryAction"
-        >
-          <ha-svg-icon .path=${mdiHistory} class="icon"></ha-svg-icon>
-          Restore Selected
-        </mwc-button>
-        ${this._snapshot.type === "full"
           ? html`
-              <mwc-button
-                @click=${this._fullRestoreClicked}
-                slot="secondaryAction"
-              >
-                <ha-svg-icon .path=${mdiHistory} class="icon"></ha-svg-icon>
-                Wipe &amp; restore
-              </mwc-button>
+              <div class="button-row">
```

Comment:
```suggestion
              <div class="button-row" slot="secondaryAction">
```

Reply:
Slots have no effect?
<https://github.com/home-assistant/frontend/pull/7677/files#diff-a125a832ada409222c511971db1296a29f700a4d9ae9fb35a403c9dd7d75f751R117>

Reply:
Why did you choose to hide the actions?

Reply:
Because it's up to 4 actions, so a simple primary/secondary does not make sense?

Reply:
That's what the `stacked` attr on the dialog is for